projects/js-packages/components/components/admin-page/index.jsx

```diff
@@ -34,7 +35,7 @@ const AdminPage = props => {
 			{ showHeader && (
 				<Container horizontalSpacing={ 5 }>
 					<Col>
-						<JetpackLogo />
+						<JetpackLogo { ...logo } />
```

Comment:
Let's avoid this as much as possible and be precise in which properties we forward

Reply:
IMO I prefer this way, basically because we could move JetpackLogo forward without caring to get back on other points to open for new props.

Reply:
No, this is an anti pattern. We even have a lint rule against it but not enforced as an error. We should enable to be errors instead

* https://vhudyma-blog.eu/react-antipatterns-props-spreading/
* https://codeburst.io/react-anti-pattern-jsx-spread-attributes-59d1dd53677f

Reply:
Ok 😄